CHANGELOG.rst

```diff
@@ -10,8 +10,17 @@ This document describes changes between each past release.
 
 - Remove pypy supports. (#1049)
 
+**Internal changes**
+
+- Permission schema children fields are now set during initialization instead of on
+  deserialization (#1046).
+- Request schemas (including validation and deserialization) are now isolated by method
+  and endpoint type (#1047).
+- Move generic API schemas (e.g TimeStamps and HeaderFields) from `kinto.core.resource.schema`
+  to a sepate file on `kinto.core.schema`.
```

Comment:
> We can deprecated them, no worries.

OK. I don't have a strong opinion about it. We can keep them too. In case we deprecate or move them, do you think we should add warning messages or we can just make it a breaking change?